### ISSUE TITLE
xdot installed but Link Map not working

### ISSUE THREAD

**Issue by teodor-a** [zim-desktop-wiki/zim-desktop-wiki#1068]
I have installed xdot according to instructions but Zim still says "• xdot - Failed" under plugin description in preferences. I thus cannot activate the plugin. GraphViz on the other hand is OK. Windows 10, installed zim from source according to instructions.

**Comment by jaap-karssenberg** [comment]
You may need to restart zim before it loads xdot successfully.

If that doesn't work, please include a bit more information, like what
operating system you are using and how you installed xdot.

**Comment by Olaf987** [comment]
I have some experience with this on the windows versions:

Using the MSYS install method, the missing xdot occurs even though I installed graphviz using pacman. I turns out you have to install the python xdot moy
dule as well using pip. this means that you have to install pip first. For 64bits:

pacman -S mingw-w64-x86_64-python-pip

And then install xdot

pip install xdot

This resolves the problem.

I just tried the windows installer (73.1) and the latest window graphviz (2.44.1). This has the missing xdot again. For now I will stick with the msys version.

**Comment by Kiskadee-dev** [comment]
I have this issue on zim 0.74.2 and windows 10.

Installed GraphViz (2.49.3) with scoop and xdot (1.2) with pip, both on path, zim fails to find them and it's not possible to enable the plugin.

**Comment by YMAir** [comment]
Same problem 
I tried to install xdot via pacman -S mingw-w64-x86_64-python-pip too 
xdot installed sucessfully , but Zim keep saying xdot - Failed
(OS windows 7 , zim 0.74.2)

**Comment by frogmouse** [comment]
Different problem, but just as frustrating.  I followed all the instructions and got failure after failure.  First MSYS2 didn't function well when I installed it into C:\Program Files: it wanted very strongly to be in C:\MSYS64.  How 1970's is that?  Then pacman kept failing until I found where it lived.  Once MSYS was up to date, pacman refused to install numpy because it couldn't find its lock file.  So I've retired from the field of battle for now, having discovered that I have a new python 3.10 in C\Python, the one I generally use in C:\Program Files\Python38, and another 3.10 in /User/(me)/AppData/Roaming/Python.  Time to go do a housecleaning and look for something other than MSYS to run GTK stuff.  Rant over.  
Obviously, this has nothing to do with the wonderfulness that is Zim.  Thanks for reading.
